onnxruntime/core/providers/cuda/shared_inc/fpgeneric.h

```diff
@@ -39,6 +46,7 @@ inline cublasStatus_t cublasGemmHelper(cublasHandle_t handle, cublasOperation_t
     onnxruntime::cuda::CublasMathModeSetter math_mode_setter(handle, CUBLAS_TENSOR_OP_MATH);
     return cublasHgemm(handle, transa, transb, m, n, k, alpha, A, lda, B, ldb, beta, C, ldc);
   }
+#endif
```

Comment:
Maybe keep the function interface untouched?

Macro within function and use ORT_UNUSED_PARAMETER(prob) to work around parameter not used problem.

Reply:
That's what I needed! thanks!